webapp/Components/ItemsGridComponent.razor

```diff
@@ -0,0 +1,41 @@
+@using System.Globalization
+
+<!-- Start Items Grid Area -->
+<section class="items-grid section custom-padding">
+    <div class="container">
+        <ItemsGridHeaderComponent/>
+        <div class="single-head">
+            <div class="row">
+                <ItemCardComponent DetailImage="assets/images/items-grid/img2.jpg"
+                                    Tag="Real Estate"
+                                    Title="Amazing Room for Rent"
+                                    Description="Blahdsakjfsjakdg sakjdhkjsag kjasgjkasgd kjaghkjasghdkj jkasghkjdf kajgkajds"
+                                    Likes=20
+                                    Cost=400
+                                    />
+                <ItemCardComponent DetailImage="assets/images/items-grid/img1.jpg"
+                                    Tag="Mobile Phones"
+                                    Title="Apple IPhone X"
+                                    Description="Blahdsakjfsjakdg sakjdhkjsag kjasgjkasgd kjaghkjasghdkj jkasghkjdf kajgkajds"
+                                    Likes=35
+                                    Cost=200
+                                    />
+                <ItemCardComponent DetailImage="assets/images/items-grid/img3.jpg"
+                                    Tag="Mobile Phones"
+                                    Title="Canon SX Powershot D-SLR"
+                                    Description="Blahdsakjfsjakdg sakjdhkjsag kjasgjkasgd kjaghkjasghdkj jkasghkjdf kajgkajds"
+                                    Likes=55
+                                    Cost=700
+                                    />
+                <ItemCardComponent DetailImage="assets/images/items-grid/img4.jpg"
+                                    Tag="Vehicles"
```

Comment:
foreach(var item in Items)
{
   <ItemGridComponent Model="@item"/>
}

public List<ItemGridComponentModel> Items { get; set; }